less/index.less

```diff
@@ -62,114 +68,244 @@ h1, h2, h3, h4, h5 {
+}
 
+// Header Styles
 header {
     width: 100%;
     display: flex;
-}
 
-header nav ul {
-    margin-top: 40px; 
-    width: 610px;
-    justify-content: space-between;
-    display: flex;
-}
+    @media @tablets {
+        flex-wrap: wrap;
+        justify-content: center;
+    }
 
-header nav ul li a {
-    text-decoration: none;
-    color: #b0b0b0;
-}
+    @media @smartphones {
+        flex-wrap: wrap;
+        justify-content: center;
+    }
 
-header nav ul li a:hover {
-    text-decoration: underline;
-    color: black;
-}
+    nav {
 
-header .logo {
-    align-self: flex-end;
-    margin: 28px 0 0 91px;
-}
+        @media @tablets {
+            width: 90%;
+        }
+
+        @media @smartphones {
+            width: 100%;
+        }
+    }
+
+    ul {
+        margin-top: 40px;
+        margin-left: 20px;
+        width: 610px;
+        justify-content: space-between;
+        display: flex;
 
-/* Call To Action (CTA) Styles */
+        @media @tablets {
+            width: 100%;
+            margin: 20px 0;
+        }
 
+        @media @smartphones {
+            width: 100%;
+            display: flex;
+            flex-direction: column;
+            margin: 0;
+            align-items: center;
+        }
+
+        li {
+
+            @media @smartphones {
+                width: 100%;
+                margin: 0;
+            }
+
+            a {
+                text-decoration: none;
+                color: #b0b0b0;
+
+                @media @smartphones {
+                    display: block;
+                    text-align: center;
+                    padding: 20px 0;
+                    font-size: 2rem;
+                    border-bottom: 1px solid gray;
+                }
+
+                &:hover {
+                    text-decoration: underline;
+                    color: black;
+
+                    @media @smartphones {
+                        background: gray;
+                    }
+                }
+            }
+
+            &:first-child {
+
+                @media @smartphones {
+                    border-top: 1px solid gray;
+                }
+            }
+        }
+    }
+
+    .logo {
+        align-self: flex-end;
+        margin: 28px 0 0 61px;
+
+        @media @tablets {
+            order: -1;
+            max-width: 100%;
+            margin: 20px 0;
+        }
+
+        @media @smartphones {
+            order: -1;
+            max-width: 100%;
+            margin: 20px 0;
+        }
+    }
+}
+
+// Call To Action (CTA) Styles
 .cta {
     display: flex;
     margin-top: 72px;
     justify-content: flex-end;
-}
 
-.cta .cta-text {
-    font-size: 72px;
-    text-align: center;
-    letter-spacing: 5px;
-    display: flex;
-    justify-content: center;
-    flex-flow: column;
-    align-items: center;
-    margin: 0 82px 0 0;
+    @media @tablets {
+        margin-top: 50px;
+        flex-flow: column;
+        align-items: center;
+    }
+
+    .cta-text {
+        font-size: 72px;
```

Comment:
Try and be consistent with your choice of units for `font-size`. You used `rem` in other sections